doc/04-Templates.md

```diff
@@ -86,10 +86,24 @@ macro in place for the actual perfdata-label:
 
     $host_name_template$.perfdata.$perfdata_label$.value
 
+You can also use wildcards. To define a wildcard, please use the following syntax:
+
+Enter the micro name, then `:` and then the syntax for the wildcard. The `:` lets the graphite module
+know that a wildcard exists here.
+
+    $micro:wildcard syntax here$
+    
+  Some Examples:
+
+    $perfdata_label:{abc,def}$
+    $perfdata_label:{a*c,de*}$
+    $perfdata_label:{a[vbn]c,def}$
+
 > **Note:**
 >
 > The name of the macro for the perfdata-label is also arbitrary. You may as
-> well use a more descriptive name such as `$disk$` for the disk check.
+> well use a more descriptive name such as `$disk$` for the disk check. The `$disk$` 
```

Comment:
The `$disk$`? Either drop the `The` or continue afterwards with `macro`: So either `$disk$ is...` or `The $disk$ macro is...`

Reply:
at the end of the sentence the same please.